lib/project.rb

```diff
@@ -82,8 +96,14 @@ def init_class_variables(options)
     @timestamp = create_timestamp
     @tag = options.tag
     @driver_path = options.path
-    @driver = find_driver
-    @engine = @config['engine']
+    if options.install
```

Comment:
The usage of engine or engine specific options here should follow object-oriented principles.
The engine object or class should provide all the needed info.

Explicit driver installation here is actually the only place where this paradigm is broken now. 
Easy to fix to renaming the operation to more generic and again taking it from the engine.

Reply:
I think we should create init_class_variables function in engine and call it. What do you think?